cloudflare/workers/backblaze.js

```diff
@@ -9,7 +9,10 @@ addEventListener('fetch', event => {
 async function handleRequest(event) {
   // only allow get requests
   if (event.request.method !== 'GET') {
-    return new Response('Method not allowed', { status: 405 })
+    return new Response('Method not allowed', {
+        status: 405,
+        headers: { 'allowed': 'GET' }
```

Comment:
Isn't it `Allow`? Or is this some library-specific name?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Allow

Reply:
Genuine typo on my part, the hands moved quicker than the brain.